lib/modules/manager/gradle/extract.ts

```diff
@@ -84,7 +84,7 @@ export async function extractAllPackageFiles(
         updateVars(gradleVars);
         extractedDeps.push(...deps);
       }
-    } catch (err) {
+    } catch (err) /* istanbul ignore next */ {
```

Comment:
Can't we somehow trigger this? How this was covered before?

Reply:
I didn't remove any testcase but with the switch to async/await `parseGradle` this somehow came to the surface.

I have no idea how to test this, to be honest, especially since there is no return or anything following the logger warning.